synapse/module_api/__init__.py

```diff
@@ -1452,6 +1452,30 @@ async def get_monthly_active_users_by_service(
             start_timestamp, end_timestamp
         )
 
+    async def lookup_room_alias(self, room_alias: str) -> Tuple[str, List[str]]:
+        """
+        Get the room ID associated with a room alias.
+
+        Added in Synapse v1.66.0.
```

Comment:
Actually I got confused with releases :facepalm: 1.64.0 is the release that came out this week, 1.65.0rc1 is scheduled for next week.

```suggestion
        Added in Synapse v1.65.0.
```